code/game/objects/effects/proximity.dm

```diff
@@ -32,7 +32,8 @@
 	host = null
 	last_host_loc = null
 	hasprox_receiver = null
-	QDEL_LIST(checkers)
+	if(!isnull(checkers))
+		QDEL_LIST(checkers)
```

Comment:
The strange thing is that QDEL_LIST has a falsy check in it, is checkers somehow being not a list?
If it runtimed before then surely it would still runtime.
if(L) { for(var/I in L) qdel(I); L.Cut(); }
The only cause where this should runtime is if checkers is not a list, in which case it will still runtime and shouldn't be a list anyway OR if a value inside checkers is null (Unless qdel(I) has a null check in it).